src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+:root {
+  --custom-color: rgba(61, 78, 97, 1);
+
+  font-family: Avenir, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.big-search {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.small-search {
+  margin: 0;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  outline: none;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: var(--custom-color);
+}
+
+.input::placeholder {
+  color: var(--custom-color);
+}
+
+.input__big {
+  height: 70px;
+
+  padding-left: 62px;
+
+  background-size: 19px 19px;
+  background-position: 26px center;
+  font-size: 16px;
+}
+
+.input__big:focus {
+  text-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.input__small {
+  height: 42px;
+
+  padding-left: 33px;
+
+  background-size: 11px 11px;
+  background-position: 13px center;
+  font-size: 14px;
+}
+
+.input__small:focus {
+  text-shadow: none;
+}
+
+.input__small:hover {
+  text-shadow: none;
+}
+
+.input__big:hover,
+.input__small:hover {
+  font-family: Avenir-Book, sans-serif;
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
```

Comment:
If both `input__big` and `input__small` implement the same style changes, maybe they should belong to the base `input`

Reply:
So I suppose I have to use CSS nesting? I'm gonna use it on the next update